documentation/INSTALL.md

```diff
@@ -83,8 +83,8 @@ novaezsiteaccessfactory_default_siteaccess_groups: ['site_groupe1', 'site_group2
 
 If you use MariaDB, the version of MariaDB must be at least version 10.2
 
-This bundle will add 1 table named: `novaez_siteaccess_factory_site_configuration` and will create 2 Content Types named
-`novaezsiteaccessfactory_home_page` and `novaezsiteaccessfactory_site_configuration`
+This bundle will add 1 table named: `novaez_siteaccess_factory_site_configuration` and will create 3 Content Types named
+`novaezsiteaccessfactory_home_page`, `novaezsiteaccessfactory_home_page_vide` and `novaezsiteaccessfactory_site_configuration`
```

Comment:
`vide` ->`empty`

Reply:
I even wonder of that name is correct, how about: `novaezsiteaccessfactory_static_home_page`

Reply:
@Plopix done for changing identifier by novaezsiteaccessfactory_static_home_page